.gitignore

```diff
@@ -27,6 +27,7 @@
 .scalastyle
 .settings
 .cache*
+.tmpBin
```

Comment:
Oh, so this is not a time-reuse device … what a bummer :-( (but thanks for the explanation ;-) )